packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js

```diff
@@ -0,0 +1,31 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import TranscribedLines from './TranscribedLines'
+
+let wrapper
+
+describe('Component > TranscribedLines', function () {
+  before(function () {
+    wrapper = shallow(<TranscribedLines />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+
+  describe('incomplete lines', function () {
+    it('should be red', function () {})
+    it('should be focusable', function () {})
+    it('should be clickable', function () {})
+  })
+
+  describe('completed lines',function () {
+    it('should be grey', function () {})
+    it('should be labelled with the consensus text', function () {})
+    it('should show the label on hover', function () {})
+    it('should be focusable', {})
+    it('should show the label on focus', function ())
+    it('should not be clickable')
```

Comment:
From the user stories, the hover/focus state is for a tooltip:

> As a user I want to hover over others’ previous markings to see a hover state explaining that these markings are other users’ markings.

Then the `consensus_text` is viewed with an onclick event to show the subtask that is disabled from editing (or something similar). 

> As a user I want to be able to read completed (greyed-out) transcriptions. 

Is there a reason you're not scaffolding this to behave this way?

Reply:
ASM has code that I think shows the consensus text on focus/hover. That’s where these specs have come from.

Reply:
The other reason for the labelling here was to add descriptive alt text for the line.

Reply:
Here's what I'm aiming for, from the ASM production collaborative workflow on this branch.
![Screenshot of the browser DOM inspector, showing each SVG line described by the consensus text for that line.](https://user-images.githubusercontent.com/59547/75242758-95fdd880-57c0-11ea-9310-e65be32d16df.png)

Help text for each line can, of course, be added via `aria-describedby` or similar.

Reply:
Simple tooltips are done but I think it would be more accessible to also show the current consensus when lines have focus.
![A focused line, showing a tooltip clarifying that the line is complete.](https://user-images.githubusercontent.com/59547/75539614-6fcf7700-5a12-11ea-87a6-73931bb749a5.png)